src/argilla/client/sdk/commons/models.py

```diff
@@ -45,6 +45,7 @@ class BaseRecord(GenericModel, Generic[T]):
     status: Optional[TaskStatus] = None
     prediction: Optional[T] = None
     annotation: Optional[T] = None
+    embeddings: Optional[Dict[str, Any]] = None
```

Comment:
Could we simplify the client embedding configuration and pass only the embedding name and the vector array as value?

Also, to be more user-friendly, could we accept NumPy arrays or pandas series for the vector value?


```python
EmbeddingVector = Union[List[float], <the-numpy-array-typing>, <the-pandas-series-typing>]
embeddings: Optional[Dict[str,EmbeddingVector]]
```

This change will add some extra logic to the SDK mapping but could simplify the user integration and engage the use of the library.